extensions/amp-autocomplete/0.1/test/test-amp-autocomplete.js

```diff
@@ -232,6 +232,52 @@ describes.realWin(
       });
     });
 
+    it('getRemoteData_() should proxy XHR to viewer', () => {
```

Comment:
This should be:

```javascript
describe('getRemoteData_()', () => {
  it('should proxy XHR to the viewer', () => {

  });
});
```

Where the feature is `getRemoteData_()` and the specific behavior being tested is "should proxy XHR to viewer".